docs/core/tools/dotnet-new.md

```diff
@@ -284,10 +284,12 @@ Each project template may have additional options available. The core templates
 
 **web**
 
-`--use-launch-settings` - Includes *launchSettings.json* in the generated template output.
+`--exclude-launch-settings` - Whether to exclude *launchSettings.json* from the generated template.
```

Comment:
wouldn't be better to just say excludes?

Reply:
In this instance I copy\pasted from the command help.